docs/devel/en/coding_conventions.rst

```diff
@@ -32,6 +32,23 @@ There is a utility method in URLUtils that will do this for you:
 
 Use it!
 
+Logging in Bundles
+------------------
+To write logs that appears in the Error Log View with different states, its recommend to use LoggingSupport-class of core bundle.
```

Comment:
Only little things:
... logs that appear ... it is recommended ...

Reply:
Thank you for the feedback! Documentation is fixed